lib/job.go

```diff
@@ -0,0 +1,9 @@
+package lib
+
+// JobMasterV2 holds the config of a job and used for failover
+type JobMasterV2 struct {
+	ID         string      `json:"id"`
+	Tp         WorkerType  `json:"type"`
+	Ext        interface{} `json:"ext"`
```

Comment:
is Ext possible to be a non-string field?

Reply:
will refine in another pr

Reply:
we should not update too much in one PR.